src/app/pages/offline-queue/offline-queue.page.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from "@angular/core";
+
+@Component({
+  selector: "app-offline-queue",
+  templateUrl: "./offline-queue.page.html",
+  styleUrls: ["./offline-queue.page.scss"]
+})
+export class OfflineQueuePage implements OnInit {
+  constructor() {}
+  stagedItems: any;
+
+  ngOnInit() {
+    this.getData();
+  }
+
+  public getData() {
+    this.stagedItems = JSON.parse(
```

Comment:
This will trigger angular bind for a millisecond. Each writes to fields of Page rewrites UI. 
It will be more efficient to simply put that to var and assign it only bellow.